openvino/conditional_compilation/CMakeLists.txt

```diff
@@ -36,6 +36,11 @@ elseif(SELECTIVE_BUILD STREQUAL "ON")
 
     target_compile_definitions(${TARGET_NAME} INTERFACE SELECTIVE_BUILD)
 
+    if (UNIX)
```

Comment:
```cmake
if(CMAKE_CXX_COMPILER_IS_GNUCXX OR
   CMAKE_CXX_COMPILER_ID MATCHES "^(Apple)?Clang$")
```

Reply:
The CMAKE_CXX_COMPILER_IS_GNUCXX  isn't defined for Cmake 3.19.3 (this version installed on my workstation) but I can use this condition: CMAKE_CXX_COMPILER_ID STREQUAL "GNU".

Reply:
FWIW, the variable is called `CMAKE_COMPILER_IS_GNUCXX`.

Reply:
https://cmake.org/cmake/help/v3.13/variable/CMAKE_COMPILER_IS_GNUCXX.html

Reply:
ok, thanks